Musicplayer.css

```diff
@@ -109,6 +121,25 @@ ul li ul li:hover{background: black}
 	text-align:center;
 }
 
+.section3{
+	background-color: white;
+	width: 85%;
+	height: 200px;
+	position: absolute;
+	z-index: 4;
+}
+
+
+
+
+.section3 h1{
```

Comment:
21/9: This is the setting for the text for of section 3 which was the div box that use to display the calculation of the total duration of all the song in the table.